components/omega/src/ocn/TendencyTerms.cpp

```diff
@@ -42,26 +44,30 @@ VelocityHyperDiffOnEdge::VelocityHyperDiffOnEdge(const HorzMesh *Mesh)
       DcEdge(Mesh->DcEdge), DvEdge(Mesh->DvEdge),
       MeshScalingDel4(Mesh->MeshScalingDel4), EdgeMask(Mesh->EdgeMask) {}
 
+WindForcingOnEdge::WindForcingOnEdge(const HorzMesh *Mesh)
+    : EdgeMask(Mesh->EdgeMask) {}
```

Comment:
```suggestion
    : Enabled(false), EdgeMask(Mesh->EdgeMask) {}
```
Following the same initialization as Bottom Drag. It is not required since we initialize when we read the yaml file, but still good coding practice.